src/BackgroundCells.js

```diff
@@ -62,7 +62,12 @@ class BackgroundCells extends React.Component {
         {range.map((date, index) => {
           let selected =  selecting && index >= startIdx && index <= endIdx;
           return (
-            <Wrapper key={index} value={date}>
+            <Wrapper
+              key={index}
+              value={date}
+              isFirstInRange={index === 0}
+              isLastInRange={index === range.length - 1}
```

Comment:
There is a reason for it. Maybe you can give me an alternative.
On our calendar when use clicks on a date we open a popover. If the day is the first one in the row popover open with arrow left aligned
![image](https://cloud.githubusercontent.com/assets/49499/26357227/eedf2ce0-3fce-11e7-9ee8-95c3da4eb042.png)

If it's `last` cell in the row it opens in this way:
![image](https://cloud.githubusercontent.com/assets/49499/26357283/1875c08c-3fcf-11e7-95d2-56e7324b2d15.png)


If it's not `first` or `last` it opens centered
![image](https://cloud.githubusercontent.com/assets/49499/26357254/0349bb6e-3fcf-11e7-9a02-fec183c35ecc.png)